src/linagora.esn.contact/app/contact/contact.service.js

```diff
@@ -1,177 +1,178 @@
+'use strict';
+
 const _ = require('lodash');
+const { ESNDavImportClient } = require('esn-dav-import-client/src');
```

Comment:
Maybe we should start doing clean stuff on our own imports. If the `esn-dav-import-client` declares its main correctly, you can remove the `/src`. For example, in the `package.json`, you must declare main like:

```
  "main": "src/index.js",
```

And then you can just do 

```js
const { ESNDavImportClient } = require('esn-dav-import-client');
```

or 

```js
import ESNDavImportClient from 'esn-dav-import-client';
```

Reply:
(this is what I did for auth module)

Reply:
Fixed.

BTW, I already declared the main in `esn-dav-import-client`